nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadService.java

```diff
@@ -135,6 +141,45 @@ public long getLength() {
     String getDownloadUrl(String storeKey);
 
     /**
+     * Gets the path and action of the URL to use to download blobs from its URI
+     *
+     * @param uri the URI of the URL to use to download blobs
+     *            ("nxfile/default/3727ef6b-cf8c-4f27-ab2c-79de0171a2c8/files:files/0/file/image.png")
```

Comment:
I'd prefer to see the examples as part of the main Javadoc and not in the `@param` section.